include/aa_pipeline_api.hpp

```diff
@@ -338,6 +338,22 @@ namespace astroaccelerate {
 			}
 		}
 
+		size_t range(){
+			return m_ddtr_strategy.range();
+		}
+
+		const int* ndms_data(){
+			return m_ddtr_strategy.ndms_data();
+		}
+
+		int tprocessed(){
```

Comment:
This should be again nSamples_in_first_chunk

Reply:
Deleted as the function is not used anymore